CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Contribution in the form of [Issues] and [PRs] are welcome.
```

Comment:
Missing links.

Reply:
I fixed the links: https://github.com/Learnosity/learnosity-sdk-ruby/commit/49bdaa704749f64b8172cab01235a90973edc4b1